torch/csrc/jit/type.h

```diff
@@ -129,7 +129,7 @@ struct TORCH_API DynamicType : public Type {
   static constexpr bool is_singleton = true;
   template<typename ... T>
   static DynamicTypePtr create( T&& ... all ) {
-    return DynamicTypePtr(new DynamicType( std::forward<T>(all)... ));
+    return DynamicTypePtr(new DynamicType( std::forward<T>(all)... )); // NOLINT(modernize-make-shared)
```

Comment:
Because it's a private constructor, I guess?

Reply:
Yes!!! It's unfortunate :/